tests/dispersytestclass.py

```diff
@@ -18,7 +25,7 @@
 # use logger.conf if it exists
 if os.path.exists("logger.conf"):
     # will raise an exception when logger.conf is malformed
-    logging.config.fileConfig("logger.conf")
+    logging.basicConfig(filename="logger.conf")
```

Comment:
@snorberhuis what issue where you having with this?

Also, better get used to split commits into different features, not a bunch of unrelated stuff.